k8s/migration/internal/controller/vjailbreaknode_controller.go

```diff
@@ -135,6 +129,13 @@ func (r *VjailbreakNodeReconciler) reconcileNormal(ctx context.Context,
 		return ctrl.Result{}, errors.Wrap(err, "failed to create openstack vm for worker node")
 	}
 
+	// Get active migrations happening on the node
+	activeMigrations, err := utils.GetActiveMigrations(vjNode.Name, ctx, r.Client)
+	if err != nil {
+		return ctrl.Result{}, errors.Wrap(err, "failed to get active migrations")
+	}
+
+	vjNode.Status.ActiveMigrations = activeMigrations
```

Comment:
After the VM is created in openstack, code control will never reach this point. Also you want to keep checking activemigrations periodically. So, move this inside the code block at 105, and add a `RequeAfter` for one minute